src/thinkspace/client/thinkspace-builder-rat/addon/components/content/sections/trat.coffee

```diff
@@ -0,0 +1,31 @@
+import ember         from 'ember'
+import base          from 'thinkspace-base/components/base'
+
+###
+# # irat.coffee
+- Type: **Component**
+- Package: **ethinkspace-builder-rat**
+###
+export default base.extend
+  # classNameBindings: ['has_questions::is-readonly']
+
+  model: null
+  type:  'trat'
+
+  manager: ember.inject.service()
+
+  question_items: ember.computed.reads 'step.trat_question_items'
+
+  has_questions: ember.computed.notEmpty 'question_items'
+
+  display_questions: ember.computed.not 'sync_assessments'
+  sync_checked:      ember.computed.reads 'sync_assessments'
+
+  actions:
+    create: ->
+      manager = @get('manager')
+      manager.add_question_item(@get('type'))
+
+    toggle_assessment_sync: (val) -> 
+      val = if val=='true' then false else true
```

Comment:
That's there because the `common/checkbox` component passes strings instead of booleans. I thought that the component rendering the `common/checkbox` was the right place to handle that. 

I'm not sure which component you're referring to, but we could pass the handling upstream or swap to the toggleProperty approach. For now I'm adding a comment to explain line 30.